domains/transfer-agreements/TransferAgreements.API/API/Controllers/TransferAgreementsController.cs

```diff
@@ -37,4 +40,24 @@ public async Task<ActionResult> Create([FromBody] CreateTransferAgreement reques
 
         return Created($"api/transfer-agreements/{result.Id}", result);
     }
+
+    [ProducesResponseType(200)]
+    [ProducesResponseType(404)]
+    [HttpGet("api/transfer-agreements")]
+    public async Task<ActionResult<TransferAgreementsResponse>> GetTransferAgreements()
+    {
+        var subject = User.FindSubjectClaim();
+
+
+        var transferAgreements = await transferAgreementRepository.GetTransferAgreementsBySubjectId(Guid.Parse(subject));
```

Comment:
Why is only agreements returned where the user is Sender? Why not where the user's CVR is receiver?

Reply:
Fixed